treelib/tree.py

```diff
@@ -616,7 +621,7 @@ def paste(self, nid, new_tree, deep=False):
         set_joint = set(new_tree._nodes) & set(self._nodes)  # joint keys
         if set_joint:
             # TODO: a deprecated routine is needed to avoid exception
-            raise ValueError('Duplicated nodes %s exists.' % list(set_joint))
+            raise ValueError('Duplicated nodes %s exists.' % list(map(text, set_joint)))
```

Comment:
else you get `"Duplicated nodes [u'A'] exists."` instead of `"Duplicated nodes ['A'] exists."`